src/cli/src/cluster/install/k8.rs

```diff
@@ -20,12 +20,31 @@ fn pre_install_check() -> Result<(), CliError> {
 
     const DEFAULT_HELM_VERSION: &'static str = "3.2.0";
 
-    if Version::parse(&version_text_trimmed) <= Version::parse(DEFAULT_HELM_VERSION) {
+    if Version::parse(&version_text_trimmed) < Version::parse(DEFAULT_HELM_VERSION) {
         return Err(CliError::Other(format!(
             "Helm version {} is not compatible with fluvio platform, please install version >= {}",
             version_text_trimmed, DEFAULT_HELM_VERSION
         )));
     }
+
+    const SYS_CHART_VERSION: &'static str = "0.1.0";
+
+    let sys_charts = helm::installed_sys_charts("fluvio-sys");
+    if sys_charts.len() < 1 {
+        return Err(CliError::Other(format!(
+            "Fluvio system chart is not installed, please install fluvio-sys first",
+        )));
+    } else {
+        let installed_chart = sys_charts.first().unwrap();
+        let installed_chart_version = installed_chart.app_version.clone();
+        if Version::parse(&installed_chart_version) < Version::parse(SYS_CHART_VERSION) {
+            return Err(CliError::Other(format!(
```

Comment:
Also perform sermver check as well not just if version is there.

Reply:
I am making a version check i.e. if version < 0.1.0, it is erroring out.

Reply:
ok.